ProcessLib/LIE/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -403,6 +404,24 @@ void HydroMechanicsProcess<GlobalDim>::initializeConcreteProcess(
             MeshLib::MeshItemType::Node, 1);
         mesh_prop_nodal_p->resize(mesh.getNumberOfNodes());
         _process_data.mesh_prop_nodal_p = mesh_prop_nodal_p;
+
+        _process_data.nodal_forces = MeshLib::getOrCreateMeshProperty<double>(
+            const_cast<MeshLib::Mesh&>(mesh), "NodalForces",
+            MeshLib::MeshItemType::Node, GlobalDim);
+        assert(_process_data.nodal_forces->size() ==
```

Comment:
For the future: We need an assertion that also covers release mode for such cheap checks.